pennylane/transforms/__init__.py

```diff
@@ -44,7 +44,9 @@
     :toctree: api
 
     ~adjoint
+    ~transforms.cancel_inverses
```

Comment:
Once we have more compilation transforms available, these should go in their own section together.